src/components/app-header.sass

```
@@ -16,19 +16,30 @@ $member-size: 18px
   font-size: 14px
 
   .left
-    flex-grow: 0
+    flex: 2
     padding: 0 $padding
     display: flex
     align-items: center
-
+    justify-content: space-between
```

```suggestion
    justify-content: flex-start
```
This would provide more of the behavior I would expect when the windows grows wider. The padding might need to be increased a little, to make the area around the spacer a little more defined.

---

Oh you are right.  That looks a lot better. Thanks!